proposals/0064-transaction-receipt.md

```diff
@@ -118,15 +118,32 @@ result in a chain split.
 
 The transaction receipt must contain the following information related to the transaction:
 
+- Version
 - Message Hash
 - Execution Status
 
 The receipt would be a structure defined as:
 
 ```rust
-pub struct TransactionReceipt {
-    pub message_hash: [u8;32],
-    pub status: u8 // 1 or 0 would determine the post execution status
+// Scalars are encoded in little-endian order
+
+const RECEIPT_VERSION_V1: u64 = 1;
+
+const RECEIPT_STATUS_SUCCESS: u8 = 0;
+const RECEIPT_STATUS_FAILURE: u8 = 1;
+
+// Size: 0x29
+struct TransactionReceipt {
+    // Offset: 0x00
+    // Must be RECEIPT_VERSION_V1
+    version: u64, // version of the receipt that starts with 1
+    
+    // Offset: 0x08
+    message_hash: [u8;32],
+
+    // Offset: 0x28
+    // Must be one of RECEIPT_STATUS_{SUCCESS,FAILURE}
+    status: u8 // 1 or 0 to determine the post execution status
 }
```

Comment:
@ripatel-fd since you told me you would be adding co-author when merging I didn't add it for this commit but will keep this in mind going forward, thank you.